sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/spring/aad/implementation/AuthorizedClientRepoTest.java

```diff
@@ -13,11 +13,13 @@
 import org.springframework.security.oauth2.client.OAuth2AuthorizedClient;
 import org.springframework.security.oauth2.client.registration.ClientRegistration;
 import org.springframework.security.oauth2.client.web.OAuth2AuthorizedClientRepository;
+import org.springframework.security.oauth2.core.AbstractOAuth2Token;
```

Comment:
Is this test class more like a unit test instead of an integration test？

Reply:
I created a PR: https://github.com/Azure/azure-sdk-for-java/pull/17857